src/v/kafka/server/group.cc

```diff
@@ -972,7 +1022,12 @@ group::join_group_stages group::add_member_and_rebalance(
 group::join_group_stages
 group::update_member_and_rebalance(member_ptr member, join_group_request&& r) {
     auto response = update_member(
-      std::move(member), r.native_member_protocols());
+      std::move(member),
+      r.native_member_protocols(),
+      r.client_id,
+      r.client_host,
+      r.data.session_timeout_ms,
+      r.data.rebalance_timeout_ms);
```

Comment:
Merge conflict due to "native_member_protocol" being a member function, before.
It is a free function in dev.